computer-vision/tools/grab_image.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+import cv2
+
+cap = cv2.VideoCapture(0)
+
+n = 0
+while(n < 10):
```

Comment:
no main?

Reply:
Same, consider this proof of concept. The python scripts were not really cleaned up yet.

Reply:
mkay, also no function to be reused, so no reason to import the file